### PR TITLE
issue#9 - print updated dates

### PR DESCRIPTION
**Pull request by mohit2agrawal**

### CODE REVIEW

File: bin/cli.js [hunk 1]
```diff
@@ -55,15 +55,25 @@ opts.nonmembers = opts.nonmembers ? opts.nonmembers.split(',') : [];
 missedIssues(opts)
   .then(issues => {
     var lastRepo = null;
+    var lastDate = null;
     issues
       .sort((a, b) => {
-        if (a.repo === b.repo) return a.id - b.id;
+        if (a.repo === b.repo){
+          if (a.updated_at === b.updated_at)
```
Comment (mcwhittemore) [inline bin/cli.js]: the name of the variable is `updated`

File: bin/cli.js [hunk 2]
```diff
@@ -55,15 +55,25 @@ opts.nonmembers = opts.nonmembers ? opts.nonmembers.split(',') : [];
...
         return a.repo < b.repo ? -1 : 1;
       })
       .map(iss => {
         if (lastRepo !== iss.repo) {
           console.log(`${lastRepo === null ? '' : '\n'}## ${iss.repo}\n`); // eslint-disable-line no-console
           lastRepo = iss.repo;
+          lastDate = null;
+        }
+        if (lastDate !== iss.updated_at){
+          console.log(`\n**${iss.updated_at}**\n`); // eslint-disable-line no-console
```
Comment (mcwhittemore) [inline bin/cli.js]: Currently this prints out `undefined` as the variable is `updated` rather than `updated_at`

**Review by mcwhittemore — DISMISSED** [review]
Thanks for this @mohit2agrawal. It's really close. A few small changes and we can merge

**Comment by mohit2agrawal** [comment]
Hi @mcwhittemore, I have made the changes.

**Review by mcwhittemore — CHANGES_REQUESTED** [review]
Mohit,

Sorry. One more thing. This currently adds date and time rather than just date. This means that most issues will have their own header that looks something like `**2017-09-12T00:59:34Z**`. The linked ticketed requested the date so this should look something like `**2017-09-12**`. This should be considered both with printing and when comparing the last date to the current one.

Thanks,
Matthew

**Comment by mohit2agrawal** [comment]
Hi @mcwhittemore 
Please check with the changed date.

**Review by mcwhittemore — APPROVED** [review]

**Comment by mcwhittemore** [comment]
Thanks @mohit2agrawal!

**Pull request merged**